src/index.js

```diff
@@ -152,6 +152,13 @@ class RedContentScript extends ContentScript {
         passwordFieldSelector,
         credentials.password
       )
+      await this.evaluateInWorker(function uncheckAndHideRememberMe() {
+        const checkBox = document.querySelector('#remember-me')
```

Comment:
You should check if the checkBox element exists and click it and hide it if it exists, with an explicit error message if not. Protect yourself or we will get an "undefined is not a function" error